authclient/MicrosoftAuth.php

```diff
@@ -10,16 +10,6 @@
  */
 class MicrosoftAuth extends Live
 {
-    /**
-     * @inheritdoc
```

Comment:
This shouldn't be removed and instead updated as suggested in https://github.com/humhub-contrib/auth-microsoft/pull/1/files#r1499595544.